packages/editor-sdk/src/components/Widgets/ModuleWidget.tsx

```diff
@@ -65,6 +99,21 @@ export const ModuleWidget: React.FC<WidgetProps<ModuleWidgetType>> = props => {
     });
   };
 
+  const modulePropertiesSpec = useMemo<JSONSchema7>(() => {
+    const obj = mapValues(module?.metadata.exampleProperties, p => {
+      let type = Types.STRING;
+      let result;
+      if (isExpression(p)) {
```

Comment:
No need to check here. `deepEval` will check whether it is expression.

Reply:
changed